src/js/views/globals/patient-modal/patient-modal_views.js

```diff
@@ -249,6 +243,11 @@ function getPatientModal(opts) {
 
   if (canEdit) {
     const type = patient.isNew() ? 'add' : 'edit';
+    const { submit_text: submitText } = Radio.request('bootstrap', 'setting', 'patient_creation_form') || {};
+
+    if (submitText && patient.isNew()) {
+      extend(opts, { submitText });
+    }
```

Comment:
My understanding is that these bugs are just with the submit button text. 

On QA currently, the submit buttons behave correctly in the edit and view patient modals. They're just showing the `patient_creation_form` setting value when they shouldn't be.